examples/stable_diffusion_v2/text_to_image.py

```diff
@@ -124,7 +124,7 @@ def main(args):
 
     # set ms context
     device_id = int(os.getenv("DEVICE_ID", 0))
-    mode = 0
+    mode = ms.context.GRAPH_MODE 
```

Comment:
if this is the only place `mode` is used, then the variable may not be needed

Reply:
Actually, most of the usage is using ms.context.GRAPH_MODE, that's why I changed this place to be consistent with other usages.